include/gatekeeper_gk.h

```diff
@@ -65,6 +65,16 @@ struct gk_measurement_metrics {
 	uint64_t tot_pkts_num_dropped;
 	/* Total size in bytes of packets dropped. */
 	uint64_t tot_pkts_size_dropped;
+	/* Total number of packets distributed to other blocks. */
+	uint64_t tot_pkts_num_distributed;
+	/* Total size in bytes of packets distributed to other blocks. */
+	uint64_t tot_pkts_size_distributed;
+	/* Total number of packets handed off to the IPv4 and IPv6 ACLs. */
+	uint64_t tot_pkts_num_acls;
+	/*
+	 * Total size in bytes of packets handed off to the IPv4 and IPv6 ACLs.
+	 */
+	uint64_t tot_pkts_size_acls;
```

Comment:
We don't need `tot_pkts_num_acls` and `tot_pkts_size_acls` since this condition can be counted on `tot_pkts_num_distributed` and `tot_pkts_size_distributed`, respectively.